src/signatureInfoView.ts

```diff
@@ -0,0 +1,312 @@
+import * as vscode from 'vscode';
+import { Renderer } from './renderer';
+
+enum UpdateMode {
+	Live = 'live',
+	Sticky = 'sticky',
+}
+
+export class SignatureInfoViewViewProvider implements vscode.WebviewViewProvider {
```

Comment:
Most of this file seems to be compied from the existing doc view code. Can you share code between these two, such as extracting common log to a base class used for docs and signature help?

Reply:
OK. Makes sense. Will do it.